src/prompters/selectPassengerCount.js

```diff
@@ -0,0 +1,25 @@
+import inquirer from 'inquirer';
+
+const isValidPassengerCount = (passengerCount) => {
+  if (!Number.isNaN(parseFloat(passengerCount))
```

Comment:
Maybe using the [`is-number` `npm` library](https://www.npmjs.com/package/is-number) will create less overhead?

Reply:
Yeah, I was just using the convention in `selectMaximumPrice.js`. Maybe it's worth using `is-number` there, too.